common/cache/domainCache.go

```diff
@@ -675,6 +677,7 @@ func (c *domainCache) buildEntryFromRecord(
 	newEntry.failoverVersion = record.FailoverVersion
 	newEntry.isGlobalDomain = record.IsGlobalDomain
 	newEntry.failoverNotificationVersion = record.FailoverNotificationVersion
+	newEntry.failoverEndTime = record.FailoverEndTime
```

Comment:
record.F or .f?

Reply:
This GetDomainResponse is defined in a different package. Have to use F